crates/quick_action_bar/src/repl_menu.rs

```diff
@@ -294,6 +294,11 @@ impl QuickActionBar {
             return div().into_any_element();
         };
 
+        let Some(worktree_id) = worktree_id_for_editor(editor.downgrade(), cx) else {
+            // todo!()
```

Comment:
Perhaps `render_kernel_selector` should return an `Option<AnyElement>` instead of this. We just shouldn't even render this if there's no worktree because of later constraints with the toolchains. A little bit too bad because it's nice to be able to have an untitled buffer be a scratch space.